app/src/main/res/layout/fragment_example.xml

```diff
@@ -1,95 +1,100 @@
 <?xml version="1.0" encoding="utf-8"?>
 <layout
 	xmlns:android="http://schemas.android.com/apk/res/android"
-	xmlns:app="http://schemas.android.com/apk/res-auto">
+	xmlns:app="http://schemas.android.com/apk/res-auto"
+	xmlns:tools="http://schemas.android.com/tools">
 
 	<androidx.constraintlayout.widget.ConstraintLayout
 		android:layout_width="match_parent"
 		android:layout_height="match_parent"
+		android:background="@color/fullWhite"
 		android:gravity="center"
-		android:orientation="vertical"
-		android:padding="@dimen/spacing_large">
+		android:orientation="vertical">
 
-		<TextView
-			android:id="@+id/title"
+		<ImageView
+			android:id="@+id/backgroundLogo"
+			android:layout_width="match_parent"
+			android:layout_height="@dimen/background_image_height"
+			android:adjustViewBounds="true"
+			android:background="@drawable/login_cover"
+			android:scaleType="centerCrop"
+			android:src="@drawable/gradient"
+			app:layout_constraintTop_toTopOf="parent" />
+
+		<ImageView
+			android:id="@+id/woloxLogo"
 			android:layout_width="wrap_content"
 			android:layout_height="wrap_content"
-			android:text="@string/example_title"
-			android:textSize="@dimen/font_medium_extra"
-			app:layout_constraintBottom_toTopOf="@id/usernameInput"
+			android:layout_marginTop="@dimen/wolox_logo_margin_top"
 			app:layout_constraintEnd_toEndOf="parent"
 			app:layout_constraintStart_toStartOf="parent"
-			app:layout_constraintTop_toTopOf="parent"
-			app:layout_constraintVertical_bias="0" />
+			app:layout_constraintTop_toTopOf="@+id/backgroundLogo"
+			app:srcCompat="@drawable/wolox_logo_large" />
 
 		<EditText
-			android:id="@+id/usernameInput"
+			android:id="@+id/email_login"
```

Comment:
Take care at the naming conventions. We are used to use camelCase instead of snake case for ids. In fact, all the other ids are in camelCase. The only things we usually save using snake_case are the strings and dimens